src/frontend/src/pages/AdminToolsPage/AdminToolsPage.tsx

```diff
@@ -15,9 +15,11 @@ const AdminToolsPage: React.FC = () => {
 
   return (
     <PageLayout title="Admin Tools">
-      <AdminToolsUserManagement />
+      {isHead(currentUser.role) && <AdminToolsUserManagement />}
       {isAdmin(currentUser.role) && <AdminToolsSlackUpcomingDeadlines />}
-      {isAdmin(currentUser.role) && <AdminToolsFinanceConfig />}
+      {(isAdmin(currentUser.role) || (isLeadership(currentUser.role) && currentUser.isFinance)) && (
```

Comment:
According to the ticket description, should we also allow both heads and leads to see `AdminToolsSlackUpcomingDeadlines` and `AdminToolsUserManagement` sections? This is how I understood when I read the ticket description. We still have role restrictions set up on setting the user's role and setting the slack deadline, so just showing two sections to heads and leads will be fine.

Reply:
we should only show the financeConfig to finance leads, any other permissions. visibility stuff should not be changed